defaults/config.js

```diff
@@ -368,4 +368,11 @@ module.exports = {
 	// @default  false
 	//
 	debug: false,
+
+	// Enables logging raw IRC messages into each server window.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	debugRaw: false,
```

Comment:
For example, I have debug logs enabled to debug connection issues on the server (connection stuff being logged to server console), and in that case I wouldn't want my client to get all the raw traffic.